tools/functions.sh

```diff
@@ -108,8 +108,7 @@ do_curl_tarball() {
     dldir="$(mktempdir)"
     mkdir "${dldir}/${cmd}"
     do_curl "${dldir}/${cmd}.tar.gz" "${url}"
-    tar -C "${dldir}/${cmd}" -xvf "${dldir}/${cmd}.tar.gz"
```

Comment:
I am not really sure why this is in there but it caused the hash to be different from what was in the checksum.

Reply:
This expands the tarball into the directory.  Your changes put the tarball in the directory w/o expanding it.